osu.Game/Skinning/LegacySkin.cs

```diff
@@ -453,11 +453,18 @@ protected override void ParseConfigurationStream(Stream stream)
 
         private Drawable? getJudgementAnimation(HitResult result)
         {
-            if (result.IsMiss())
-                return this.GetAnimation("hit0", true, false);
-
             switch (result)
             {
+                case HitResult.Miss:
+                    return this.GetAnimation("hit0", true, false);
+
+                case HitResult.LargeTickMiss:
+                    return this.GetAnimation("slidertickmiss", true, false);
+
+                case HitResult.ComboBreak:
```

Comment:
`ComboBreak` is not used by any osu! judgement. If this was targeting classic mod, this should be `SmallTickMiss`.

Even fixing the above is not sufficient, as the pool for `SmallTickMiss` is not registered. Applying the following helps that:

```diff
diff --git a/osu.Game.Rulesets.Osu/UI/OsuPlayfield.cs b/osu.Game.Rulesets.Osu/UI/OsuPlayfield.cs
index c94057cf6d..4dd38bc0ad 100644
--- a/osu.Game.Rulesets.Osu/UI/OsuPlayfield.cs
+++ b/osu.Game.Rulesets.Osu/UI/OsuPlayfield.cs
@@ -74,6 +74,7 @@ public OsuPlayfield()
                              case HitResult.Meh:
                              case HitResult.Miss:
                              case HitResult.LargeTickMiss:
+                             case HitResult.SmallTickMiss:
                              case HitResult.IgnoreMiss:
                                  return true;
                          }
diff --git a/osu.Game/Skinning/LegacySkin.cs b/osu.Game/Skinning/LegacySkin.cs
index 270abe2849..eb9b3635c4 100644
--- a/osu.Game/Skinning/LegacySkin.cs
+++ b/osu.Game/Skinning/LegacySkin.cs
@@ -461,7 +461,7 @@ protected override void ParseConfigurationStream(Stream stream)
                 case HitResult.LargeTickMiss:
                     return this.GetAnimation("slidertickmiss", true, false);
 
-                case HitResult.ComboBreak:
+                case HitResult.SmallTickMiss:
                 case HitResult.IgnoreMiss:
                     return this.GetAnimation("sliderendmiss", true, false);
 
```

But even _that_ doesn't help, as this happens if you do it:

![osu_2023-12-25_13-33-57](https://github.com/ppy/osu/assets/20418176/ce9ea491-3c9d-4382-a301-33b5676a4e91)

Note stacked big miss and small sliderend miss. This is because

https://github.com/ppy/osu/blob/4fc1691a578ad16256591d33e8a614c9de9426da/osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs#L53

at which point, I'm lost at what to do.

Reply:
Classic mod display doesn't matter too much, because slider ends when it's turned on are broken anyway, so i think just dropping the line is fine?

Reply:
Sure. If you're ok with showing tick misses but not tail misses then that works.